doc/python_api/rst/bge_types/bge.types.SCA_PythonJoystick.rst

```diff
@@ -37,7 +37,7 @@ base class --- :class:`PyObjectPlus`
       * up:[0.0, -1.0, ...]
       * down:[0.0, 1.0, ...]
 
-   .. attribute:: hatValues
+   .. attribute:: hatValues (Deprecated. Use activeButtons instead)
```

Comment:
To add links to the doc you could write:
 Use :data:`activeButtons` instead

Other than that feel free to merge :)